### PR TITLE
Update self managed changelog

### PR DESCRIPTION
**Pull request by amyblais**
Clarify details on inactive server email notifications feature.

### CODE REVIEW

**Review by svelle — APPROVED** [review]
Looks good to me.

Unrelated to this specific PR: Do we have any docs on how to enable/disable feature flags?

**Comment by coltoneshaw** [comment]
@svelle - I don't think so. Generally the feature flags are more hidden to enable/disable.

File: source/install/self-managed-changelog.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Mattermost v6.5.0 contains medium severity level security fixes. [Upgrading](htt
  - The post menu is now only rendered on the root post on hover over.
  - Updated a library used for storing drafts and other data in browser storage.
  - Enabled performance telemetry tracking for production deployments not running in developer mode. This telemetry tracking is disabled when telemetry is toggled off.
- - Inactive server email notifications will now be sent to users occasionally if they haven't interacted with their server for 100 hours or more.
+ - Inactive server email notifications will now be sent to System Admins occasionally if they haven't interacted with their server for 100 hours or more.  Inactivity is determined by reviewing all activity on the server. This feature can also be disabled using the ``Mattermost_FeatureFlags_EnableInactivityCheckJob`` feature flag.
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:64]:
```suggestion
 - Inactive server email notifications will now be sent to System Admins occasionally if they haven't interacted with their server for 100 hours or more. Inactivity is determined by reviewing all activity on the server. This feature can also be disabled using the ``Mattermost_FeatureFlags_EnableInactivityCheckJob`` feature flag.
```

**Review by coltoneshaw — APPROVED** [review]

**Comment by coltoneshaw** [comment]
> I'm wondering something similar -- Will System Admins know how to disable this feature flag? Or is this something they'd reach out to Support for?

Generally, they wouldn't know how to enable them. We have a bit of environment variable documentation [here](https://docs.mattermost.com/configure/configuration-settings.html#environment-variables)./

File: source/install/self-managed-changelog.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Mattermost v6.5.0 contains medium severity level security fixes. [Upgrading](htt
  - The post menu is now only rendered on the root post on hover over.
  - Updated a library used for storing drafts and other data in browser storage.
  - Enabled performance telemetry tracking for production deployments not running in developer mode. This telemetry tracking is disabled when telemetry is toggled off.
- - Inactive server email notifications will now be sent to users occasionally if they haven't interacted with their server for 100 hours or more.
+ - Inactive server email notifications will now be sent to System Admins occasionally if they haven't interacted with their server for 100 hours or more. Inactivity is determined by reviewing all activity on the server. This feature can also be disabled using the ``Mattermost_FeatureFlags_EnableInactivityCheckJob`` feature flag.
```
Comment (dontoisme) [inline source/install/self-managed-changelog.md:64]: The only change I would recommend for 64 is changing from "... occasionally if they haven't interacted with their server..." to "... if there have been no telemetry events on their server..."

I'm not certain the feature flag can be disabled by the admin. @marianunez when you are back from vacation can you answer?

**Pull request merged**